resources/bluima_sentence_detector_genia/src/main/java/ch/epfl/bbp/nlp/res/sentence/GeniaResource.java

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.bbp.nlp.res.sentence;
+
+import ch.epfl.bbp.nlp.ModelResource;
+
+/**
+ * Genia model for sentence splitter annotator
+ */
+public class GeniaResource implements ModelResource {
+    public String getResourcePath() {
+        return "/SentDetectGenia.bin.gz";
```

Comment:
don't we want to put a version (that corresponds to the pom's version) here in the file name? e.g. 'SentDetectGenia-0.1.bin.gz'